docs/autogeneration/formatter.py

```diff
@@ -80,3 +81,25 @@ def convert_df_to_table(
     df["sphinx"] = "\t* - " + df.sphinx
 
     return "\n".join(list(df.sphinx))
+
+
+def camel_case_split(identifier):
+    matches = finditer(
+        ".+?(?:(?<=[a-z])(?=[A-Z])|(?<=[A-Z])(?=[A-Z][a-z])|$)", identifier
+    )
+    return [m.group(0) for m in matches]
+
+
+def extract_docstring_info_from_evaluator(object):
```

Comment:
...is there a way to integrate this into Github workflow for linting?

E.g. on push, this function runs. If it "fails" (throws an error, but maybe something more comprehensive), then that signifies someone wrote a bad evaluator docstring and needs to go fix it

Reply:
Created a ticket for this stuff: https://credo-ai.atlassian.net/browse/DSP-315